crates/test_utils/wasm/wasm_workspace/the_incredible_halt/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "test_wasm_the_incredible_halt"
+version = "0.0.1"
+authors = [ "thedavidmeister", "thedavidmeister@gmail.com" ]
+edition = "2021"
+
+[lib]
+name = "test_wasm_the_incredible_halt"
+crate-type = [ "cdylib", "rlib" ]
```

Comment:
This will need an example for the integrity zome. Check the other test wasms to see what I mean.